src/validate.ts

```diff
@@ -26,7 +26,7 @@ function innerValidate(data: any, schema: SchemaLike) {
 }
 
 export function validate(
-  schema: SchemaLike,
+  schema: Schema,
```

Comment:
I suggest you stick to `SchemaLike`. You could then use `compile` internally

Reply:
Done.